api/tests/organizational-entities/integration/infrastructure/repositories/certification-center-for-admin.repository.test.js

```diff
@@ -23,11 +23,14 @@ describe('Integration | Organizational Entities | Infrastructure | Repository |
       const certificationCenterId = 1;
       const certificationCenterName = 'CertificationCenterName';
       const certificationCenterType = 'SCO';
+      const userId = databaseBuilder.factory.buildUser().id;
+      await databaseBuilder.commit();
 
       const center = databaseBuilder.factory.buildCertificationCenter({
         id: certificationCenterId,
         name: certificationCenterName,
         type: certificationCenterType,
+        createdBy: userId,
       });
 
       const certificationCenterForAdmin = new CenterForAdmin({
```

Comment:
Ne pas construire d'objet CertificationCenter, mais fournir uniquement les attributs nécessaires à sa création
```suggestion
      const certificationCenterForAdmin = new CenterForAdmin({
        center: { name: certificationCenterName, type: certificationCenterType },
      });
```